### PR TITLE
Refactor constants in Ceph Plugin

### PR DESCRIPTION
**Pull request by bipuladh**
Fixing the organization of the constants in OCS Plugin

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts [hunk 1]
```diff
@@ -50,56 +65,24 @@ export enum CreateStepsSC {
...
+});
 
-export const KMSMaxFileUploadSize = 4000000;
-export const KMSConfigMapName = 'ocs-kms-connection-details';
-export const KMSConfigMapCSIName = 'csi-kms-connection-details';
-export const KMSSecretName = 'ocs-kms-token';
+export const OCS_DEVICE_REPLICA = Object.freeze({
+  '2': '2-way',
+  '3': '3-way',
+});
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts:82]: This is a pool configuration we can create a new file for this.

**Comment by cloudbehl** [comment]
LGTM

@afreen23 can you take a look once.

File: frontend/packages/ceph-storage-plugin/src/constants/index.ts [hunk 2]
```diff
@@ -1,60 +1,8 @@
...
-  READY = 'Ready',
-  PROGRESSING = 'Progressing',
-}
+export * from './network';
+export * from './storage-pool-const';
+export * from './kms';
+export * from './tooltips';
+export * from './common';
+export * from './disks';
+export * from './pool';
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/constants/index.ts:8]: Why are we keeping index.ts file ?

**Pull request merged**